examples/index.js

```diff
@@ -1,12 +1,15 @@
 /* eslint no-console: "off" */
 
 const express = require('express');
```

Comment:
minor: you need to keep the 1 line space otherwise the eslinter will complain: "[eslint] Expected empty line after require statement not followed by another require. (import/newline-after-import)"